src/app_manager/app_manager.py

```diff
@@ -201,6 +204,11 @@ def __init__(
         # end_time = time.time()
         # rospy.logerr('total time: {}'.format(end_time - start_time))
 
+        rospy.loginfo("Initializing default launcher")
+        self._default_launch = roslaunch.parent.ROSLaunchParent(
+            rospy.get_param("/run_id"), [], is_core=False, show_summary=False)
```

Comment:
`show_summary` keyword arg is not found in kinetic, which causes CI error.

Reply:
You can copy remove the keyword arg in kinetic.

Reply:
You are right
[kinetic documentation](https://docs.ros.org/en/kinetic/api/roslaunch/html/roslaunch.parent.ROSLaunchParent-class.html)
[melodic documentation](https://docs.ros.org/en/melodic/api/roslaunch/html/roslaunch.parent.ROSLaunchParent-class.html)

Am removing it altogether instead of forking between before-kinetic and after-melodic since it is not a significant change:
with show_summary=False:
```
started roslaunch server http://...
auto-starting new master
process[master]: started with pid [4964]
ROS_MASTER_URI=http://localhost:11311
setting /run_id to this
process[rosout-1]: started with pid [4978]
started core service [/rosout]
```
Without show_summary keyword (show_summary=True):
```
started roslaunch server http://...

SUMMARY
========

PARAMETERS
 * /rosdistro: melodic
 * /rosversion: 1.14.12

NODES

auto-starting new master
process[master]: started with pid [5251]
ROS_MASTER_URI=http://localhost:11311
setting /run_id to this
process[rosout-1]: started with pid [5267]
started core service [/rosout]
```